src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Consent.cshtml

```diff
@@ -0,0 +1,82 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.ConsentModel
+@{
+    ViewData["Title"] = "Permission to share details";
+}
+
+@section Back {
+    <a href="/ProfessionalReferral/ConnectFamilyToServiceStart?id=@Model.Id&name=@Model.Name" class="govuk-back-link">Back</a>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        @if (!Model.ValidationValid)
+        {
+            <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" data-module="govuk-error-summary">
+                <h2 class="govuk-error-summary__title" id="error-summary-title">
+                    There is a problem
+                </h2>
+                <div class="govuk-error-summary__body">
+                    <ul class="govuk-list govuk-error-summary__list">
+                        <li>
+                            <a href="#service-name" data-testid="enter-service-name-error">Select whether you have permission from the family to share details.</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        }
+
+        <h1 class="govuk-heading-l">
+            Permission to share details
+        </h1>
+        <form method="post">
+            <input asp-for="@Model.Name" type="hidden">
+            <input asp-for="@Model.Id" type="hidden">
+            <p>You must have permission from the family to give their details to the service.</p>
+            <p class="govuk-body">Those details include:</p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>who to contact in the family</li>
+                <li>ways to contact the family</li>
+                <li>why the family needs help </li>
+            </ul>
+            <p>You can come back and connect the family to the service once you have permission.</p>
+            <div class="govuk-form-group">
+                <fieldset class="govuk-fieldset">
```

Comment:
We should follow the example of the [inline radio](https://design-system.service.gov.uk/components/radios/), e.g.  use a legend, add the appropriate aria attributes etc.

Reply:
Taken from the [inline radio](https://design-system.service.gov.uk/components/radios/)